Formula/bashtop.rb

```diff
@@ -0,0 +1,45 @@
+class Bashtop < Formula
+  include Language::Python::Virtualenv
+
+  desc "Bash only* alternative to top, htop, vtop, glances"
+  homepage "https://github.com/aristocratos/bashtop"
+  url "https://github.com/aristocratos/bashtop/archive/v0.9.16.tar.gz"
+  sha256 "836a80864f85dd41fb818f30c2be789931055e45ce1a5842e739e2ec28531367"
+  head "https://github.com/aristocratos/bashtop.git"
+
+  depends_on "bash"
+  depends_on "coreutils"
+  depends_on "gnu-sed"
```

Comment:
Apologies if this has already been discussed, but I think it would make sense to add the [optional dependency `osx-cpu-temp`](https://github.com/aristocratos/bashtop#optionals-for-additional-stats) as a dependency to get a more feature rich experience.

```suggestion
  depends_on "gnu-sed"
  depends_on "osx-cpu-temp"
```

Reply:
My understanding is that homebrew-core disallows "optional" dependencies. I tried this and the `brew audit --strict <formula>` test rejects optional dependencies when present. Is that a mistake?

Reply:
No,homebrew optional dependencies are disallowed. Upstream makes this optional but the suggestion wouldn't make it optional in homebrew.